src/ray/raylet/node_manager.h

```diff
@@ -95,6 +95,17 @@ class NodeManager : public rpc::NodeManagerServiceHandler,
   /// Get the port of the node manager rpc server.
   int GetServerPort() const { return node_manager_server_.GetPort(); }
 
+  /// Preprocess each request from the raylet client.
+  /// Handling three common parts for requests. Printing log, checking heartbeat
+  /// and checking worker being killed are included.
```

Comment:
nit: it might be better to rephrase this like:

Preprocess request from raylet client. This takes care of common processing for each request, such as logging, checking heartbeat and whether worker is killed.